lightning/src/ln/outbound_payment.rs

```diff
@@ -433,8 +435,8 @@ impl OutboundPayments {
 		}
 	}
 
-	fn send_payment_internal<K: Deref, F>
-		(&self, route: &Route, payment_hash: PaymentHash, payment_secret: &Option<PaymentSecret>,
+	fn send_payment_internal<K: Deref, F>(
```

Comment:
nit: I'd also move the close `)` to the next line here and all the following methods as you did in the methods before this (ec55730959103b71417d23d3a77011ca7d7d4ee5).